README.md

```diff
@@ -21,5 +21,13 @@ Check full features and entities involved on the [wiki home page](https://github
 8. Create database: `rake db:create`
 9. Run migrations: `rake db:migrate`
 10. Seed database with initial data: `rake db:seed`
+This creaete five users with 
```

Comment:
So, can you add these things in the README as well? Or a better thing would be to list test user accounts in different section of README. Whatever seems good to you!